docs/src/examples/components/Chat/Types/ChatExample.shorthand.tsx

```diff
@@ -8,101 +8,106 @@ const janeAvatar = {
 
 const items: ShorthandCollection<ChatItemProps> = [
   {
-    message: (
-      <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
-    ),
+    message: {
+      content: 'Hello',
+      author: 'John Doe',
+      timestamp: 'Yesterday, 10:15 PM',
+      mine: true,
+    },
     contentPosition: 'end',
     attached: 'top',
     key: 'message-id-1',
   },
   {
-    message: (
-      <Chat.Message content="I'm back!" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
-    ),
+    message: {
+      content: "I'm back!",
+      author: 'John Doe',
+      timestamp: 'Yesterday, 10:15 PM',
+      mine: true,
+    },
     contentPosition: 'end',
     attached: true,
     key: 'message-id-2',
   },
   {
-    message: (
-      <Chat.Message
-        content={{
-          content: (
-            <div>
-              What do you think about <a href="#">www.goodFood.com</a>?
-            </div>
-          ),
-        }}
-        author="John Doe"
-        timestamp="Yesterday, 10:15 PM"
-        mine
-      />
-    ),
+    message: {
+      content: (
+        <div>
+          What do you think about <a href="#">www.goodFood.com</a>?
+        </div>
+      ),
+      author: 'John Doe',
+      timestamp: 'Yesterday, 10:15 PM',
+      mine: true,
+    },
     contentPosition: 'end',
     attached: 'bottom',
     key: 'message-id-3',
   },
   {
     gutter: <Avatar {...janeAvatar} />,
-    message: <Chat.Message content="Hi" author="Jane Doe" timestamp="Yesterday, 10:15 PM" />,
+    message: {
```

Comment:
Consider that this is getting wrapped by some other element that the user now no longer has shorthand access to.  They should be able to add props and styles to this hidden area.  Our APIs shouldn't ever hide an area from the user.